nf_core/lint.py

```diff
@@ -729,10 +657,6 @@ def check_version_consistency(self):
         if self.config.get('process.container', ''):
             versions['process.container'] = self.config.get('process.container', '').strip(' \'"').split(':')[-1]
 
-        # Get version from the TRAVIS_TAG env var
-        if os.environ.get('TRAVIS_TAG') and os.environ.get('TRAVIS_REPO_SLUG', '') != 'nf-core/tools':
```

Comment:
See above, needs updating to work with GitHub actions

Reply:
See https://github.com/nf-core/tools/pull/502/commits/7ca2f22f8911385aecf2df23ecd19a26979b98f1